File: src/main/java/it/contrader/dto/ProdottoDTO.java
package it.contrader.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor

public class ProdottoDTO {
	
	private Long id;
	
	private String name;
	
	private String description;
	
	private float price;
	
	private int priority;
	
	private WishListDTO wishlist;

}

File: src/dto/prodottodto.ts
import { UserDTO } from './userdto';
import { WishListDTO } from './wishlistdto';
import { CategoryDTO } from './categorydto';


export class ProdottoDTO {
    
    category_fkId: number;

    description: string;

    id: number;

    image: string;

    name: string;

    price: number;

    priority: number;

    proprietario_id: number;
    
    wishlist_fkId: number;
    
    constructor(id?: number, name?: string,price?:number){
        this.id = id;
        this.name = name;
        this.price=price;
    }
}